docs/bots-guide.md

```diff
@@ -27,18 +27,15 @@ issue, or submit a pull request to share your ideas!
 
 # The contrib_bot system
 
-Zulip's bot system resides in the `contrib_bots` directory.
+Zulip's bot system resides in the `api` directory.
 
  **Note: There exists an additional directory called `bots`. This directory does *not* contain normal bots,
  but rather unpolished integrations.**
 
-The `contrib_bots` directory structure looks like the following:
+The `api` directory structure looks like the following:
```

Comment:
Here, we don't want to outline the whole `api` directory (which contains alot of other, non-bot related stuff), but only`bots_api`.